.github/workflows/clients-dotnet-build.yml

```diff
@@ -132,3 +134,15 @@ jobs:
           dotnet run
           "
 
+  bors_clients_dotnet:
+    name: bors/clients/dotnet
+    needs:
+      - build
+      - prepare_validation_tests
+      - run_validation_tests
+      - run_validation_tests_on_containers
+    runs-on: ubuntu-latest
+    steps:
+      - name: CI succeeded
+        working-directory: ./
+        run: exit 0
```

Comment:
These are basically grouping steps to avoid listing a million of entries in bors.toml, right? It feels like a good hack here, though naively it seems that having so much CI jobs is the problem, I wonder if we can compress that somehow, using matricies or some such. Definitely out of scope for the PR.

Reply:
Yes, exactly for that reason. Their getting started guide recommends a collecting step if you're using matrix builds too.

+1 on cleaning up the steps in a separate PR - the client ones aren't too bad, but `test_and_build.yml` has a lot of steps that could really just be a test suite (eg, all the `fuzz_`)

Reply:
Actually, it seems there's perhaps a better way to do that? You can use regexes in bors: 


>Patterns are written using SQL LIKE syntax. For example, this status directive will match any CI job that starts with ci/gitlab/:

```
status = [ 'ci/gitlab/%' ]
```

Reply:
Ah, if the docs recommend collecting step, that's the way to go!